README.md

```diff
@@ -67,6 +67,24 @@ The module gives the ability to select a specific operating system on ordering o
 
 Create a configurable option named `Operating System`. Add options with the following convention: `osId|osName`. This will take priority over selected operating system in product module settings.
 
+#### Plan params
+
+The module gives the ability to select specific plan params such as `VCPU`, `Memory`, `Disk`, `Total traffic limit monthly` (for KVM and VZ), `VCPU Units`, `VCPU Limit`, `IO Priority`, `Swap` (for VZ).
+This is done in the form of configurable options.
+
+![VCPU](./docs/option-vcpu.png)
+
+Create a configurable option named one of: `VCPU`, `Memory`, `Disk`, `Total traffic limit monthly`, `VCPU Units`, `VCPU Limit`, `IO Priority`, `Swap`. Add options with the following convention: `planParamValue|planParamTitle`. This will take priority over plan params of selected plan in product module settings.
+
+`planParamValue` for `Memory` and `Swap` set in MiB (e.g. `4096|4 GiB`).
+
+`planParamValue` for `Disk` set in GiB (e.g. `20|20 GiB`).
+
+`planParamValue` for `Total traffic limit monthly` set in units which are specified in Total Traffic limit option of the plan selected in Module Settings for a product in SolusIO (e.g. `1|1 GiB` if Total Traffic limit unit is `GiB` or `1024|1 Gib` if Total Traffic limit unit is `MiB`).
+`Total traffic limit monthly` option can only be used if the plan selected in Module Settings for a product has Total Traffic limit enabled in SolusIO.
```

Comment:
`Total traffic limit monthly` option can only be used if the plan selected in the Module Settings of a product has the `Total Traffic limit` option enabled in SolusIO.